torch/csrc/distributed/rpc/init.cpp

```diff
@@ -103,12 +103,12 @@ PyObject* rpc_init(PyObject* /* unused */) {
           &ProcessGroupAgent::sync,
           py::call_guard<py::gil_scoped_release>());
 
-  module.def("_init_rref_context", [](std::shared_ptr<RpcAgent> agent) {
-    RRefContext::initInstance(std::move(agent));
+  module.def("init_rpc_agent", [](std::shared_ptr<RpcAgent> agent) {
```

Comment:
Is this intended to be exposed to the user or private? If private, should we prefix with `_`?

Reply:
This should be private. Users shouldn't need to know RpcAgent.